AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1442,20 +1428,21 @@ - (void)__setNeedsDisplay
 
 // These private methods ensure that subclasses are not required to call super in order for _renderingSubnodes to be properly managed.
 
-- (void)__layout
+- (void)__layoutSublayers
 {
   ASDisplayNodeAssertMainThread();
   ASDN::MutexLocker l(__instanceLock__);
-  CGRect bounds = self.bounds;
-
-  [self measureNodeWithBoundsIfNecessary:bounds];
-
+  CGRect bounds = _threadSafeBounds;
+  
   if (CGRectEqualToRect(bounds, CGRectZero)) {
     // Performing layout on a zero-bounds view often results in frame calculations
     // with negative sizes after applying margins, which will cause
     // measureWithSizeRange: on subnodes to assert.
     return;
   }
+    
+  [self measureNodeWithBoundsIfNecessary:bounds];
+  _pendingDisplayNodeLayout = nullptr;
```

Comment:
The method above has a case where it does NOT apply the layout - if another transition is in progress / or a parent is transitioning (the hierarchy state?)

Is it correct to nullptr the _pendingLayout in ALL cases?  If so, shouldn't we put the = nullptr inside the method measureNodeWithBoundsIfNecessary?